app/models/ht_approval_request.rb

```diff
@@ -2,9 +2,18 @@
 
 class HTApprovalRequest < ApplicationRecord
   self.primary_key = 'id'
+  # "Active" requests that may be subject to further action
   scope :not_renewed, -> { where(renewed: nil) }
+  # "Inactive" requests that are only of historical interest
+  scope :renewed, -> { where.not(renewed: nil) }
   scope :for_approver, ->(approver) { where(approver: approver).order(:sent, :received, :renewed) }
-  scope :for_user, ->(user) { where(userid: user) }
+  scope :for_user, ->(user) { where(userid: user).order(:sent, :received, :renewed) }
```

Comment:
It looks like `for_user` is only currently used in `most_recent` below, so I'm not sure we need the order here.

Reply:
I'd rather have the order be explicit and mirroring  `for_approver` rather than sorta random, even if it is never used.

Reply:
👍